tool/planet/start.go

```diff
@@ -791,6 +788,22 @@ func copyResolvFile(cfg utils.DNSConfig, destination string, upstreamNameservers
 	return nil
 }
 
+func generateHosts() []utils.HostEntry {
+	hosts := []utils.HostEntry{
+		{IP: "127.0.0.1", Hostnames: "localhost localhost.localdomain localhost4 localhost4.localdomain4"},
+		{IP: "::1", Hostnames: "localhost localhost.localdomain localhost6 localhost6.localdomain6"},
+	}
+
+	if utils.OnGCEVM() {
+		hosts = append(hosts, utils.HostEntry{
+			IP:        "169.254.169.254",
```

Comment:
Is this a commonly used IP address for a metadata server?

Reply:
Yea, off hand both AWS and GCE use this IP for the metadata server. I don't know about other envs off hand.